nvflare/fuel_opt/README.md

```diff
@@ -0,0 +1,31 @@
+# Optional dependency module nvflare.fuel_opt
+
+The nvflare/fuel_opt is the module with components or files which **optional** dependencies.
+That means there are dependencies are not required to be installed when user type
+ ```
+     pip install nvflare
+ ```
+* you will need to install the required python package separately or use
+
+```
+  pip install nvflare[core_opt]
+```
+to install optional dependency for "core" module.  
+
+* If you are interested in CONFIG package only 
+
+```
+  pip install nvflare[CONFIG]
+```
+CONFIG is part of the core, so core_opt will include optional packages for config and other cores.  
+
+* to install optional dependency for application level module, use
+
+```
+  pip install nvflare[apt_opt]
+```
+to install all optional dependency, use
+
+```
+  pip install nvflare[all]
+```
```

Comment:
this information is good to have.
but should we put it here or the README.md in the root ?